addons/account/models/account_analytic_distribution_model.py

```diff
@@ -32,11 +33,34 @@ def _get_default_search_domain_vals(self):
 
     def _get_applicable_models(self, vals):
         applicable_models = super()._get_applicable_models(vals)
-        return applicable_models.filtered(lambda m:
-            not m.account_prefix or vals.get('account_prefix', '').startswith(m.account_prefix)
+
+        # Regex pattern to split by either ';' or ','
+        delimiter_pattern = re.compile(r'[;,]\s*')
+
+        return applicable_models.filtered(lambda model:
+            not model.account_prefix or
+            any((vals.get('account_prefix', '') or '').startswith(prefix) for prefix in delimiter_pattern.split(model.account_prefix))
         )
 
     def _create_domain(self, fname, value):
         if fname == 'account_prefix':
             return []
         return super()._create_domain(fname, value)
+
+    # To be able to see the placeholder when creating a record in the list view, need to depends on a field that has a
+    # value directly, analytic precision has a default.
+    @api.depends('analytic_precision')
```

Comment:
I could maybe force the compute in a default get, but i think just putting a depends is enough, what do you think ?

Reply:
I prefer putting one on create_date in general to be clear that it's in creation